subsys/can/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+if(CONFIG_CAN_STACKS)
+  zephyr_interface_library_named(CAN_STACKS)
+
+  zephyr_library()
+  zephyr_library_sources(common.c)
+  zephyr_library_link_libraries(CAN_STACKS)
+
+
+  add_subdirectory_ifdef(CONFIG_CANOPEN    ./canopen)
```

Comment:
Another question: the word CAN is very generic and at least I get confused sometimes when reading the code and comments. Should we call it CANBUS instead?